.github/workflows/shellcheck.yml

```diff
@@ -26,8 +26,5 @@ jobs:
       - uses: actions/checkout@v4
       - name: Run ShellCheck
         uses: ludeeus/action-shellcheck@master
-        with:
-          severity: error
-          additional_files: "catppuccin.tmux"
```

Comment:
`          additional_files: "catppuccin.tmux"`
This line should have been kept, my mistake.